imap_processing/cdf/config/imap_codice_global_cdf_attrs.yaml

```diff
@@ -16,196 +16,196 @@ instrument_base: &instrument_base
 # L1a
 imap_codice_l1a_hskp:
     <<: *instrument_base
-    Data_type: L1A_hskp->Level-1A Housekeeping Data
+    Data_type: L1A_hskp>Level-1A Housekeeping Data
```

Comment:
These changes remove a `-` to avoid a SPDF API error in the `Logical_source` check